scripts/drs_file_access_examination.py

```diff
@@ -0,0 +1,240 @@
+"""Data Repository Service (DRS) DRS File Access Examination.
+
+(Exported from a jupyter notebook, but rearrange into a python scricpt)
+
+In this notebook, we get all the DRS urls from the data table and check
+if we can access the file each DRS url points to.
+
+This notebook will:
+1. Grab all DRS urls from the data table
+2. Resolve each DRS url and check our access to the file
+3. Create a report with all the DRS urls and our access level to each of them
+
+Background:
```

Comment:
I love this detailed description and background! great job!

Reply:
Thank you!